### ISSUE TITLE
Dead link.

### ISSUE THREAD

**Issue by Strages16** [itchio/itch.io#1543]
I purchased access to a game called 'Evil Never Dies', and was sent a link to download it. However the link is non-functional. It takes me to Mega, but then a message comes up that "The file you are trying to download is no longer available". So, the developer gets my money, but I get nothing. I would like you to either get the game developer to fix this, or refund my money. Thanks.

**Comment by Blueberryy** [comment]
Yeah it could be great if itch add "Dead" marking for moderators so people don't waste money on untrusted developers

**Comment by VerzatileDev** [comment]
Seems like its a scam, you are best of contacting support@itch.io , but itchi.io themselves are not liable and generally can not help with it. So for that I would say if the purchase was made with paypal, aim to refund it or talk to their support chat, and then contact itchio support in relation to the refund.
